bedrock/firefox/views.py

```diff
@@ -443,7 +443,13 @@ def hello(request):
     }
 
     if (waffle.switch_is_active('firefox-hello-2016')):
-        return l10n_utils.render(request, 'firefox/hello/index-2016.html')
+        variant = request.GET.get('v', '')
+        if variant in ['b']:
+            template = 'firefox/hello/index-2016-{0}.html'.format(variant)
```

Comment:
hmm, you're right of course... I started with the same code running the current plugincheck test, which has four variants, and for a while I believed there were going to be two for the desktop page so I kinda locked onto that notion and never thought to simplify it for a single variant, even for the others. 

Happy to update for cleanliness if you think it's worthwhile, but then maybe best not to go messing with it and risk further delay.